Index: ac-helpers/abi-glib-gtk.m4
===================================================================
RCS file: /cvsroot/abi/ac-helpers/abi-glib-gtk.m4,v
retrieving revision 1.24
diff -u -r1.24 abi-glib-gtk.m4
--- ac-helpers/abi-glib-gtk.m4	21 Aug 2005 15:30:30 -0000	1.24
+++ ac-helpers/abi-glib-gtk.m4	13 Nov 2006 22:43:18 -0000
@@ -62,7 +62,7 @@
 	AC_PATH_X
 	AC_PATH_XTRA
 
-	GTK_LIBS="$GTK_LIBS $X_LIBS $X_PRE_LIBS -lX11 $X_EXTRA_LIBS"
+	GTK_LIBS="$GTK_LIBS $X_LIBS -lX11 $X_EXTRA_LIBS"
 	AC_SUBST(GTK_LIBS)
 ])
 
